components/server/src/server.rs

```diff
@@ -152,11 +153,13 @@ pub fn run_tikv(config: TiKvConfig) {
         }};
     }
 
-    if !config.raft_engine.enable {
-        run_impl!(RocksEngine)
-    } else {
-        run_impl!(RaftLogEngine)
-    }
+    dispatch_api_version!(config.storage.api_version(), {
```

Comment:
please merge the master, here has some changes

Reply:
Done. PTAL~